meson.build

```diff
@@ -251,7 +251,7 @@ if get_option('use_slirp') # disabled by default
 endif
 
 if get_option('enable_debugger') != 'none'
-  curses_dep = dependency('ncurses') # or pdcurses on Windows
+  curses_dep = dependency('curses') # use the new 'curses' for msys2 compatibility
```

Comment:
According to the meson documentation, this came available in meson 0.54, which is our minimum version, so should be fine on all platforms.

Reply:
Thanks for highlighting this!